server/src/auth/service.js

```diff
@@ -0,0 +1,33 @@
+const UserModel = require('../user/model')
+const bcrypt = require('bcryptjs')
+const jwt = require('jwt-simple')
+const cfg = require('../../config/auth')
+
+exports.login = (req, res) => {
+  // check if the body contains an email and a password
+  if (req.body.email && req.body.password) {
+    // body assignments
+    let email = req.body.email
+    let password = req.body.password
+    // find user by email
```

Comment:
Useless comment; And as a general comments idea, when you are in the middle of the block and you put a comment, make a new line before, so it can be readable ;)